polaris/polaris/deposit/views.py

```diff
@@ -146,15 +148,15 @@ def interactive_deposit(request):
     # Validate query parameters: account, asset_code, transaction_id.
     account = request.GET.get("account")
     if not account:
-        return render_error_response("no 'account' provided")
+        return render_error_response("no 'account' provided", content_type="text/html")
```

Comment:
is there a way to do this at a higher level rather than putting content_type="text/html" everywhere?

Reply:
Answered in your final comment below